plugins/Text Formatter-3cd1d9a8e92e4210a0fd30d1d65bb664.json

```
@@ -0,0 +1,12 @@
+{
+    "ID": "3cd1d9a8e92e4210a0fd30d1d65bb664",
+    "Name": "Text Formatter",
+    "Description": "Retrieve text from the clipboard and edit it using several custom scripts",
+    "Author": "Koalhack",
+    "Version": "0.0.1",
+    "Language": "javascript",
+    "Website": "https://github.com/Koalhack/Flow.Launcher.Plugin.Formatter",
+    "UrlDownload": "https://github.com/Koalhack/Flow.Launcher.Plugin.Formatter/releases/download/v0.0.1/Flow.Launcher.Plugin.Formatter.zip",
+    "UrlSourceCode": "https://github.com/Koalhack/Flow.Launcher.Plugin.Formatter",
+    "IcoPath": "https://cdn.jsdelivr.net/gh/Koalhack/Flow.Launcher.Plugin.Formatter@main/icon/app.png",
```

```suggestion
"IcoPath": "https://cdn.jsdelivr.net/gh/Koalhack/Flow.Launcher.Plugin.Formatter@main/icon/app.png"
```
workflows are failing due to this trailing comma